src/test/kotlin/net/maizegenetics/phgv2/cli/AgcCompressTest.kt

```diff
@@ -0,0 +1,171 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.testing.test
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.lang.Exception
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.LocalDate
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+class AgcCompressTest {
```

Comment:
Are you planning on adding tests which verify the sequence coming out matches the input FASTA sequence?  Right now you are just checking that it ran without error, Correct?

Reply:
I'm checking more than that.  

Currently it checks that it ran without error; that the assemblies.agc file exists and contains the sample names (from the fasta file names) that are expected; that when appending to an existing assemblies.agc file, the old file is written to a backup.  That the old file contains just the old samples, and that the new assemblies.agc file contains both the original and the appended samples.  ANd it checks if you try to append to an existing file, but all samples are already in the existing file, then nothing is updated.

We can check the fastas, but what is the priority here?  This is a 3rd party vendor's code.  We need to know we can call it and it is creating the files we expect.

My thoughts are we get the pipeline working with exhaustive testing on new code, but adequate testing on 3rd party software.  We can add additional test later if it is deemed necessary.

Reply:
I think we also need to make sure the FASTA sequence coming out matches what we expect.  This should be a very quick test to set up(in comparison to alignment or even imputation tests).  If you extract out sequence from a region, it needs to match what was into the original fasta file.  I agree this is a third party tool, but the basic unit test would be helpful.

Reply:
I am going to wait and add this to the next branch, which I'll start as soon as this is merged..

This branch is only dealing with creating the agc compressed files.  The next branch (once this is committed) will have the command you mentioned today at scrum (I thought you put the example in slack, but I don't see it). That branch will deal with commands that pull data out of the agc compressed file.

It makes more sense to me to add the the sequence checks in the branch that is pulling sequences.